packages/react-tooltip/src/components/Tooltip/index.ts

```diff
@@ -1,5 +1,12 @@
 export * from './Tooltip';
-export * from './Tooltip.types';
+// Explicit exports to ommit TooltipCommons
+export type {
+  TooltipProps,
+  TooltipSlots,
+  TooltipState,
+  OnVisibleChangeData,
+  TooltipTriggerProps,
+} from './Tooltip.types';
```

Comment:
nit: Order these alphabetically
```suggestion
export type {
  OnVisibleChangeData,
  TooltipProps,
  TooltipSlots,
  TooltipState,
  TooltipTriggerProps,
} from './Tooltip.types';
```